docs/docs/spec/metrics/metrics-examples.md

```diff
@@ -16,13 +16,13 @@ Some metrics and endpoints are very straightforward, but having a reference for
 ## Prometheus Metrics
 
 ### Economics metrics
-* `eigen_fees_earned_total{token="ETH"}`
-* `eigen_slashing_incurred_total{token="ETH"}`
-* `eigen_total_balance_total{token="ETH"}`
+* `eigen_fees_earned_total{token="ETH", unit="wei", strategy="eigenDA"}`
+* `eigen_slashing_status{avs="eigenDA"}`
+* `eigen_registered_stakes{token="ETH", unit="wei", strategy="eigenDA"}`
```

Comment:
remove eigenDA. see previous comment.

Reply:
let's remove token and strategy labels, and replace them with `quorum`, which is the AVS registry contract notion. stakes in a quorum represent some weighted combination of shares in eigenlayer strategies.

Reply:
Yes, since that's the only metric that is not eigenlayer contracts related and only avs registry related. Let's give the quorum example name something like "ethLST" (to represent a quorum that accepts any of the eth LSTs). Another quorum could be "beaconChainEth" if there's room for another example somewhere.

Reply:
Done [d0e911](https://github.com/NethermindEth/el-node-spec/pull/58/commits/3d0e911b83f03ccb7740f16dddcf4ed43ee53e72)